### PR TITLE
fix: improve its documentation page

### PR DESCRIPTION
**Pull request by Olanetsoft**
Revamp the ITS page.

- [x] Fix inconsistent hyperlinks/style formats 
- [x] Fix auto redirects to new ITS path
- [x] Navigation update with Card-like option for ITS features
Screenshot 2023-11-21 at 15 51 34

- [x] Create a new interchain token
Screenshot 2023-11-21 at 15 51 44

- [x] Upgrade existing token
Screenshot 2023-11-21 at 15 51 54
- [x] Add the updated code snippets
- [x] Add ITS v1 update

### CODE REVIEW

File: src/pages/dev/send-tokens/interchain-tokens/intro.astro [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+import Section from "../../../../layouts/Section.astro";
+import { Callout } from "../../../../../src/components/callout"
+
+const title = "What is Interchain Tokens?";
```
Comment (Foivos) [inline src/pages/dev/send-tokens/interchain-tokens/intro.astro:5]: "What are Interchain Tokens?"

File: src/pages/dev/send-tokens/interchain-tokens/create-token.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    - Name
+    - Symbol
+    - Decimals
+    - Amount to mint
+- You can also click the advanced option where you can add an account as `Token Minter` and `Salt` value; otherwise, it gets prefilled with the deployer(connected account) address and a random, uniquely generated salt value
+- Select additional chains for your token's availability and optionally add the token amount to mint on each selected chain.
+
+**Congratulations!** Your Interchain Token is now accessible on multiple blockchain networks. This allows for seamless interaction with your token on any of these networks, utilizing familiar methods like send, transfer, and approve, just as with any standard ERC-20 token.
+
+Furthermore, your token is categorized as a [Interchain Token](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainTokenStandard.sol). This classification includes the `interchainTransfer` method, allowing you to transfer it between blockchains.
```
Comment (Foivos) [inline src/pages/dev/send-tokens/interchain-tokens/create-token.mdx:27]: I would prefer something like "Furthermore, your token follows the [Interchain Token Standard]. This means that users can call the `interchainTransfer` method on the token itself to transfer between blockchains.

File: src/pages/dev/send-tokens/interchain-tokens/create-token.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    bytes calldata metadata,
+    uint256 gasValue
+) external payable
+```
+
+## Create a new custom Interchain Token(s)
+
+If you want more features than the Interchain Token provides, you can create a [custom](../../reference/glossary/#custom-token) token. You may want a customized token to customize specific minting policies, ownership structures, rate limits, or other bespoke functionalities.
+
+To create a new Interchain Token:
```
Comment (Foivos) [inline src/pages/dev/send-tokens/interchain-tokens/create-token.mdx:55]: The steps below describe how to deploy an InterchainToken, not a custom token. To deploy a custom token the user needs to
- deploy their custom ERC20, which should include mint/burn implementations
- use `deployTokenManager` with the correct params (mint/burn as the type) on the ITS
- make sure that the service can call mint/burn on the token.
- Do the same on any other chains the user wants to have their token exist at, using the same deployer address and salt.

File: src/pages/dev/send-tokens/interchain-tokens/upgrade-tokens.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    address originalTokenAddress,
+    string calldata destinationChain,
+    uint256 gasValue
+) external payable returns (bytes32 tokenId)
+```
+
+If pre-mint is needed, you must make ERC20 approval to the Interchain Token Factory contract. When tokens are moved from the origin chain to another chain, the token will be locked on the origin chain and minted on the destination chain. If you moved tokens directly from one non-origin chain to another, the token would be burned on the source chain and minted on the destination chain.
+
+
+## Link deployed tokens on multiple chains into Interchain Token(s)
```
Comment (Foivos) [inline src/pages/dev/send-tokens/interchain-tokens/upgrade-tokens.mdx:57]: This is what custom tokens do indeed, with regards to the previous comment.

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Please remove the _meta file you created, please remove the `-old` file you created

**Review by ffe9f8 — APPROVED** [review]

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**